src/components/shared/TreeView.js

```diff
@@ -672,6 +681,11 @@ class TreeView<DisplayData: Object> extends React.PureComponent<
             className="treeViewBody"
             role="tree"
             ariaLabel="Call tree"
+            // this attribute exposes the current active child element,
+            // while keeping focus on the parent (call tree)
+            ariaActiveDescendant={
+              selectedNodeId ? `treeViewRow-${selectedNodeId}` : null
+            }
```

Comment:
As a quick note, in the Marker Table, I get `treeViewRow--1` when there's no selection. But this is beause I made a mistake in a previous patch and I'll fix it quickly :) (normally selectedNodeId is null when it's not selected)

Reply:
I fix this in https://github.com/firefox-devtools/profiler/pull/1811.